### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by k-rahul21**

### CODE REVIEW

File: components/navigation/navigation.html [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+              <ul class="list-subtopic list-non-bullet">
+                <li>
+                  <a href="/Get Started/introduction.html">Introduction</a>
+                </li>
+              </ul>
+              </div>
+  
+              <div class="nested-list1">
+                  <div class="list-topic font-bold">
+                      <a href="#">Components</a>
```
Comment (gadgetfather) [inline components/navigation/navigation.html]: You forgot to add link here

File: components/navigation/navigation.html [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+                          <a href="/components/image/image.html">Image</a>
+                      </li>
+                      <li>
+                          <a href="/components/input/input.html">Input</a>
+                      </li>
+                      <li>
+                        <a href="/components/navigation/navigation.html">Navigation</a>
+                      </li>
+                      <li>
+                          <a href="#">List</a>
```
Comment (gadgetfather) [inline components/navigation/navigation.html:81]: here also add link

File: components/navigation/navigation.html [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+                    </ul>
+                </nav>
+           </div>
+
+           <pre>
+             <code class="code-snippet">
+
+                <nav class="navigation-bar left-aligned">
+                    <div class="left-nav">
+                      <a class="font-bold" href="">Brand Name</a>
```
Comment (gadgetfather) [inline components/navigation/navigation.html]: here also href is empty

File: components/navigation/navigation.html [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+           <div class="description wrapper-description">
+                Spaced out navigation has the brand logo and the navigation links on the extreme ends of the navbar.
+           </div>
+
+          <div class="example-div">
+
+            <div class="component-showcase">
+                <nav class="navigation-bar spaced-around">
+                    <div class="left-nav">
+                      <a class="font-bold" href="">Brand Name</a>
```
Comment (gadgetfather) [inline components/navigation/navigation.html:144]: here as well href is empty

**Review by abhij1607 — COMMENTED** [review]
Good practices followed, Nice class nomenclature!
Can be merged

File: components/navigation/navigation.html [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+                    </ul>
+                </nav>
+           </div>
+
+           <pre>
+             <code class="code-snippet">
+
+                <nav class="navigation-bar spaced-around">
+                    <div class="left-nav">
+                      <a class="font-bold" href="">Brand Name</a>
```
Comment (gadgetfather) [inline components/navigation/navigation.html]: Here also you forgot to add href link

**Review by gadgetfather — CHANGES_REQUESTED** [review]

**Review by Hardiegogo — COMMENTED** [review]
Code looks good.
Just add some media queries to make your app responsive.

**Pull request merged**